pallets/accounts/src/mock.rs

```diff
@@ -138,17 +170,52 @@ impl pallet_proxy::Config for Test {
 	type AnnouncementDepositFactor = ConstU32<1>;
 }
 
-impl Config for Test {
-	type Event = Event;
-	type WeightInfo = ();
-	type EthChainId = ConstU64<5>; // goerli
-	type EthConfirmations = ConstU64<1>;
-	type EthFragContract = ();
-	type Threshold = ConstU64<1>;
-	type AuthorityId = pallet_frag::crypto::FragAuthId;
-}
-
 pub fn new_test_ext() -> sp_io::TestExternalities {
 	let t = frame_system::GenesisConfig::default().build_storage::<Test>().unwrap();
-	t.into()
+
+	let mut ext = sp_io::TestExternalities::new(t);
+
+	ext.execute_with(|| System::set_block_number(1)); // if we don't execute this line, Events are not emitted from extrinsics (I don't know why this is the case though)
+
+	ext
+}
+
+pub fn new_test_ext_with_ocw() -> (
+	sp_io::TestExternalities,
+	Arc<RwLock<PoolState>>,
```

Comment:
oh wow, where did you find this out? nice

Reply:
https://github.com/paritytech/substrate/blob/master/frame/examples/offchain-worker/src/tests.rs#L208